src/Uno.UI/UI/Xaml/Controls/ListViewBase/NativeListViewBase.Android.cs

```diff
@@ -314,5 +317,60 @@ bool ILayoutConstraints.IsHeightConstrained(View requester)
 
 			return this.IsHeightConstrainedSimple() ?? (base.Parent as ILayoutConstraints)?.IsHeightConstrained(this) ?? false;
 		}
+
+		public override bool Fling(int velocityX, int velocityY)
+		{
+			// Workaround for inverted ListView receiving fling velocity in the opposite direction
+			// See: https://issuetracker.google.com/u/1/issues/112385925
+			velocityX = Recalibrate(velocityX, _deltaX);
+			velocityY = Recalibrate(velocityY, _deltaY);
+
+			return base.Fling(velocityX, velocityY);
+
+			int Recalibrate(int value, float? hint)
+			{
+				// note: the opposite sign should be used to recalibrate the velocity
+				return hint.HasValue && hint != 0
```

Comment:
so since we check the sign, we _should_ be futureproof even if android fixes their bug in android Q or later, correct?